Enterprise-CMCS/macpro-mdct-mfp
# set up route

### Description

PR to set up the /export routes for WP and SAR PDFs

this does not cover any PDF creation or downloading, just the route

### Related ticket(s)


CMDCT-207

---
### How to test

- log in as user or admin and skip to the Review and Submit page of the SAR or the WP
- click Review PDF
- a new tab should open for the /export route and the component should say what report the PDF page is for



### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
